### DISCUSSION TITLE
Callback for WriterExceptions

### DISCUSSION THREAD

**Discussion by fazleskhan** [JoshClose/CsvHelper#1788] [General]
I am writing +40k records out from a badly formatted set of user data. Instead of using Linq-to-Object to materialize the data, which I have found too slow, I am using Linq/LinqKit to pass the IEnumerable result directly to the csvhelper writer. Execution time is down to 14sec but the writer fails when it encounters bad user data. Is there a way to register an ExceptionCallback when writing out records like there is for readers. I would like to not have to individually handle every error condition in my Linq/LinqKit expression query and instead just log the error.

**Comment by JoshClose** [comment]
What is "bad user data"? What do you mean by it fails?
